Documents/API.md

```diff
@@ -318,6 +318,54 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
 }
 ```
 
+    * `POST` - Provides verification for macro edits
```

Comment:
__Minor__: There looks like some odd formatting issue [here](https://github.com/jmarin/hmda-platform/blob/b1256c7dcae1c40c6d306b2f24902daa96a29797/Documents/API.md) ... below  the `/institutions/<institution>/filings/<period>/submissions/<submissionId>/edits/<syntactical|validity|quality|macro>`